.github/workflows/lint.yml

```diff
@@ -0,0 +1,40 @@
+name: Lint
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+permissions:
+  checks: write
+  contents: write
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node: [16]
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Setup node env
+        uses: actions/setup-node@v2.1.2
+        with:
+          node-version: ${{ matrix.node }}
+
+      - name: Install Node.js dependencies
+        run: npm ci
```

Comment:
And pbbly let's use smth like:
```yml
      - name: Install dependencies
        run: npm install
```
Also like at  [`main`](https://github.com/ITISHub/itishub-frontend/blob/master/.github/workflows/main.yml), to have consistent `.yml` workflows

Reply:
Pbbly it may cause Lint pilpline fail at current PR, as I think